packages/starknet-snap/src/ui/fragments/FeeTokenSelector.tsx

```diff
@@ -47,6 +48,12 @@ export const FeeTokenSelector: SnapComponent<FeeTokenSelectorProps> = ({
           </SelectorOption>
         </Selector>
       </Field>
+      {error ? null : (
```

Comment:
it look weird when the error message is not because of no funds


as i mentioned, it is better to swtich use a separate error component, and display the error msg all in one?
wdyt?

{Boolean(amount.error) && <Text color="error">{amount.error}</Text>}

Reply:
keep both error message